### DISCUSSION TITLE
Willing to pay someone to build a custom WebDAV windows desktop client

### DISCUSSION THREAD

**Discussion by PrimeDominus** [lookfirst/sardine#498] [General]
We need something fairly simple. Need a desktop client that mounts our WebDAV server connections as a local drive securely.

We've been using MountainDuck, which is quite good, but it has way more features than we need for our purpose, and it forces us to restart Windows after each update.

We're currently too busy to work on this ourselves. I guess it hasn't been an urgent priority for us since we already have MountainDuck, but it's something we do need, and our team hasn't had a lot of experience building Windows desktop clients.

Please comment if you're interested in working on this and would like to discuss it further.

No need to share contact details publicly here. Based on your comment, if we feel you're potentially a good fit, I can create a private GitHub repo and invite you to it, and we can use that to share details privately.

**Comment by dkocher** [comment]
I would recommend that you reach out to [us](support@mountainduck.io) to discuss a tailored white label for your organization.

**Reply by lookfirst** [reply]
Agreed!

**Reply by PrimeDominus** [reply]
I should have known members of iterate GmbH are involved with sardine.

Having had a look at the code for cyberduck, it looks like (I could be wrong) you have built a custom Windows installer and included non-java things (e.g. .Net) to get it all working. We're thinking along the lines of having a jar that we could use with something like [install4j](https://www.ej-technologies.com/products/install4j/overview.html) to create our installer and manage auto-updates.

Happy to discuss this, but we really need something super simplified compared to the super awesome kitchen sink MountainDuck provides, plus we would need access to the code as well.
